manifest.json

```diff
@@ -37,19 +37,13 @@
   },
 
   "background": {
-    "scripts": [
-      "public/addon-build/background.js"
-    ]
+    "scripts": ["public/addon-build/background.js"]
```

Comment:
Why are we changing this file?

Reply:
hmm, looks like my linter did this for me.

Reply:
Reverted. Turns out I can "save without autoformatting" in vscode!